activerecord/lib/active_record/railtie.rb

```diff
@@ -98,6 +98,22 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
+    initializer "active_record.asynchronous_queries_executor" do
```

Comment:
This initializer may be totally unnecessary, or too soon, but that's an example of the kind of simplified configuration we could offer. For simple users setting a single max concurrency is probably plenty enough of control.